pkgs/tools/admin/zbctl/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "zbctl";
+  version = "8.0.6";
+
+  builder = ./builder.sh;
```

Theres no need to use `builder` here, please use the phases instead. 

```suggestion
  dontConfigure = true;
  dontBuild = true;

  installPhase = ''
    runHook preInstall

    mkdir -p $out/bin
    cp $src $out/bin/zbctl
    chmod +x $out/bin/zbctl

    runHook postInstall
  '';

---

Wasn't sure which one was cleaner, but this makes sense